ember-cli-build.js

```diff
@@ -4,20 +4,26 @@ const EmberApp = require('ember-cli/lib/broccoli/ember-app');
 const StaticSiteJson = require('broccoli-static-site-json');
 const BroccoliMergeTrees = require('broccoli-merge-trees');
 
-const jsonTree = new StaticSiteJson(`content/ember/v2`, {
+const contentTypes = ['ember/v1', 'ember/v2', 'ember-data/v2', 'ember-cli/v2'];
```

Comment:
I would change the name of this from `contentTypes` to `contentFolders`, it's clearer as to what it is (i.e. it doesn't actually change the type)